libs/langchain/pyproject.toml

```diff
@@ -373,6 +373,10 @@ extended_testing = [
  "xata",
 ]
 
+scheduled_testing = [
```

Comment:
will this be identical to the integration_test group?

Reply:
maybe asymptotically, but practically speaking we've got plenty of integration tests atm that shouldn't be run